.travis.yml

```diff
@@ -42,6 +42,10 @@ matrix:
     - php: 5.6
       env: DB=mysql
     - php: 5.6
-      env: DB=mysql;TC=lint
+      env: DB=mysql TC=lint
+    - php: 5.6
+      env: DB=mysql CORE_BRANCH=stable10
+    - php: 5.6
+      env: DB=mysql TC=lint CORE_BRANCH=stable10
```

Comment:
``CORE_BRANCH=stable10`` is redundant here.
Do you intend to run any matrix entries against core ``master`` ?